.travis.yml

```diff
@@ -50,6 +50,14 @@ jobs:
     env:
     - MB_PYTHON_VERSION=3.9
     - PLAT=i686
+  - arch: arm64
+    os: linux
+    services: docker
+    env:
+    - MB_PYTHON_VERSION=3.9
+    - MB_ML_VER=2014
```

Comment:
ARM support is only available on manylinux 2014; more info in [PEP 600](https://peps.python.org/pep-0600/)